README.md

```
@@ -1,3 +1,75 @@
+마
+
+- 선 한 칸 이동 + 대각선 한 칸 이동
+- 다른 기물이 이동 경로 사이에 있으면 이동 불가
+
+---
+
+사 & 궁
+
+- 모든 위치 이동 가능
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)